types/jqwidgets/index.d.ts

```diff
@@ -0,0 +1,4896 @@
+// Type definitions for jqwidgets-scripts
```

Comment:
Looks like our lint rules are disabled for this package. Can you enable the lints in our `tslint.json` and fix up the version string to be a `MAJOR.MINOR` version (e.g. `8.3`)?

Also, it looks like this containing folder was incorrectly named. jqwidgets doesn't exist on npm, but jqwidgets-scripts does.